File: Praktikum6/Bahasa C/PRAK601-2010817210023-MuhammadAuliaAkbar.c
#include <stdio.h>

int main()
{
    int baris, kolom;
    printf("Masukan Ukuran Matriks (Baris x Kolom): ");
    scanf("%d %d", &baris, &kolom);
    int Matriks[baris][kolom];

    for (int i = 0; i < baris; i++)
    {
        for (int j = 0; j < kolom; j++)
        {
            scanf("%d", &Matriks[i][j]);
        }
    }
    for (int i = 0; i < baris; i++)
    {
        for (int j = 0; j < kolom; j++)
        {
            printf("%d ", Matriks[i][j]);
        }
        printf("\n");
    }
    return 0;
}

File: Praktikum6/Bahasa Python/PRAK601-2010817210023-MuhammadAuliaAkbar.py
# Meminta input ukuran matriks
baris = int(input("Masukkan jumlah baris: "))
kolom = int(input("Masukkan jumlah kolom: "))

# Membuat matriks kosong
matriks = []

# Meminta input elemen matriks
for i in range(baris):
    baris_matriks = [int(input(f"Masukkan elemen baris {i + 1}, kolom {j + 1}: ")) for j in range(kolom)]
    matriks.append(baris_matriks)

# Menampilkan matriks
for baris in matriks:
    for elemen in baris:
        print(elemen, end=" ")
    print()
